.eslintrc.json

```diff
@@ -20,7 +20,8 @@
     "import/no-unresolved": "off",
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/222334239-b8620794-5297-4355-a803-f051feac8957.png)

**[Optional]**

- [ ] It would be nice if you make your features more like features :smile: to make your readme file descriptive and professional :+1: It's because the feature added doesn't sound like a feature because a feature might be

* Add a book
* Remove a book